src/model/document.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2022 spdx contributors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from datetime import datetime
+from typing import List, Optional
+
+from src.model.actor import Actor
+from src.model.annotation import Annotation
+from src.model.external_document_ref import ExternalDocumentRef
+from src.model.extracted_licensing_info import ExtractedLicensingInfo
+from src.model.file import File
+from src.model.package import Package
+from src.model.relationship import Relationship
+from src.model.snippet import Snippet
+from src.model.version import Version
+
+
+class CreationInfo:
+    creators: List[Actor]  # min. 1 element
```

Comment:
Why do we have a comment concerning validation here?

Reply:
It relates to our typing: In all other cases (I believe), I used `List` when a property is optional (cardinality `0..*`), with the intention to have an empty list if nothing is specified in the document. But here, the type `List` has a different meaning.
Not quite sure about it - would you suggest just removing it and just checking it in validation?

Reply:
Yes, I would propose checking this in validation. A code comment won't stop users from putting an empty list there.

Reply:
I was not suggesting not checking it in validation. This was just meant as an "extended type hint". But we can remove it